changelog/fix16997.dd

```diff
@@ -0,0 +1,19 @@
+-preview=intpromote switch is now set by default.
+
+integer promotions now follow C integral promotions rules consistently
+
+It affects operands of type byte, ubyte, short, ushort, char, and wchar. The operands are promoted to int before the operator is applied. The result type will now be int.
+
+The values computed will be different for some values of the operands:
+
+All values of - and ~ applied to ubyte, ushort, char, and dchar operands will change.
+
+The values of ``-byte(-128)`` and ``-short(-32768)`` will change.
+
+To revert to the old behavor rewrite it as the following:
+
+op b as typeof(b)(op int(b)) to preserve the old behavior.
```

Comment:
This sentence is poorly written. There should be a line in between, and the transformation given is quite vague. Use emphasis to denote the pattern being transformed, and give a concrete example

Reply:
Done